src/linux/py_proc.h

```diff
@@ -68,6 +68,21 @@ union {
 
 
 // ----------------------------------------------------------------------------
+static Elf64_Addr
+_py_proc_get_base64(Elf64_Ehdr * ehdr, void * elf_map)
```

Comment:
Could you please rename this to `_get_base_64` in accordance with the contribution guidelines for the project. 

**Reason** the `_py_proc` prefix is to be used when the first argument is a pointer to an instance of the `py_proc_t` object. Otherwise, this is just a local helper. As for the `base64` into `base_64`, the former could be a bit misleading as it might be confused with the encoding of the same name.

Reply:
OK.